include/greshunkel.h

```diff
@@ -39,7 +39,6 @@ typedef struct greshunkel_ctext {
  * xXx *sub_ctext=If this var is a sub context, this will hold the sub context for the variable. xXx
  */
 typedef union greshunkel_var {
-	const unsigned int fuck_gcc : 1; /* This tricks GCC into doing smart things. Not used. */
```

Comment:
I think this should have been left as it represents a mature and objective viewpoint on the current status of open source C compilers.